roles/zookeeper-docker/tasks/start-zookeeper-data-container.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Create ZooKeeper data container
+  docker_container:
+    name: "zookeeper_data_{{ inventory_hostname_short }}"
```

Comment:
Use my_id for constructing docker name (it would allow for more docker containers on the same host machine).